ca_gazebo/worlds/empty.world

```diff
@@ -1,42 +1,13 @@
-<?xml version="1.0"?>
-<sdf version="1.6">
+<?xml version="1.0" ?>
+<sdf version="1.5">
   <world name="default">
     <!-- A global light source -->
     <include>
       <uri>model://sun</uri>
     </include>
-
     <!-- A ground plane -->
     <include>
       <uri>model://ground_plane</uri>
     </include>
-
```

Comment:
Why did you delete this? Are the default values?

Reply:
I took another `.world` file from another project, everything is working like it should now.